source/conf.py

```diff
@@ -14,7 +14,11 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.mathjax', 'investspec']
+extensions = [
+    'sphinx.ext.mathjax',
+    'sphinx_reredirects',
```

Comment:
Do we need `sphinx_intl` here as well?

Reply:
No, `sphinx_intl` isn't a sphinx extension, just a command line tool.